MahApps.Metro/Behaviours/DatePickerTextBoxBehavior.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace MahApps.Metro.Behaviours
+{
+    using System.Windows.Controls;
+    using System.Windows.Controls.Primitives;
+    using System.Windows.Interactivity;
+    using MahApps.Metro.Controls;
+
+    public class DatePickerTextBoxBehavior : Behavior<DatePickerTextBox>
+    {
+        protected override void OnAttached()
+        {
+            base.OnAttached();
+            AssociatedObject.TextChanged += OnTextChanged;
+        }
```

Comment:
@punker76 any suggestions how to improve it?

WeakEventManager<DatePickerTextBox, TextChangedEventArgs>.AddHandler(AssociatedObject, TextBoxBase.TextChangedEvent.Name, OnTextChanged);

is only available for NET45